src/main-thread/commands/offscreen-canvas.ts

```diff
@@ -3,27 +3,46 @@ import { TransferrableKeys } from '../../transfer/TransferrableKeys';
 import { MessageType } from '../../transfer/Messages';
 import { CommandExecutor } from './interface';
 import { OffscreenCanvasMutationIndex } from '../../transfer/TransferrableMutation';
+import { NumericBoolean } from '../../utils';
 
 export function OffscreenCanvasProcessor(workerContext: WorkerContext): CommandExecutor {
   return {
     execute(mutations: Uint16Array, startPosition: number, target: RenderableElement): number {
       if (target) {
-        const offscreen = (target as HTMLCanvasElement).transferControlToOffscreen();
-        workerContext.messageToWorker({
-          [TransferrableKeys.type]: MessageType.OFFSCREEN_CANVAS_INSTANCE,
-          [TransferrableKeys.target]: [target._index_],
-          [TransferrableKeys.data]: offscreen, // Object, an OffscreenCanvas
-        }, [offscreen]);
+        const canvas = target as HTMLCanvasElement;
+        if (typeof canvas.transferControlToOffscreen === 'function') {
```

Comment:
We shouldn't be sending down a `OFFSCREEN_CANVAS_INSTANCE` request if the function isn't implemented.